src/sources/h3-query-source.ts

```diff
@@ -30,7 +33,7 @@ type UrlParameters = {
 };
 
 export const h3QuerySource = async function (
-  options: H3QuerySourceOptions
+  options: Omit<H3QuerySourceOptions, 'spatialDataType'>
```

Comment:
Primarily `H3QuerySourceOptions` represents the public-facing option types of the `h3QuerySource` function. Since users would never pass the 'spatialDataType' prop here, I think we can remove it from the 'H3QuerySourceOptions' definition.

(And similar for other sources)

Reply:
Okay, if we do this, where do we define the `spatialDataType` prop then?
Because the `spatialDataColumn` prop is defined in `QuerySourceOptions` and `TableSourceOptions`

Reply:
Other thing I was considering was moving this prop definition to `WidgetBaseSourceProps`

Reply:
That's perfect, I'm working on a second proposal for this, but seems easier to understand

Reply:
Second proposal has been merged, I'm going to mark this as resolved